tajo-core/src/main/java/org/apache/tajo/engine/planner/physical/SeqScanExec.java

```diff
@@ -228,26 +237,15 @@ public Tuple next() throws IOException {
       return null;
     }
 
-    Tuple tuple;
-    Tuple outTuple = new VTuple(outColumnNum);
-
-    if (!plan.hasQual()) {
-      if ((tuple = scanner.next()) != null) {
-        projector.eval(tuple, outTuple);
-        outTuple.setOffset(tuple.getOffset());
-        return outTuple;
-      } else {
-        return null;
-      }
-    } else {
-      while ((tuple = scanner.next()) != null) {
-        if (qual.eval(tuple).isTrue()) {
-          projector.eval(tuple, outTuple);
-          return outTuple;
-        }
-      }
-      return null;
+    while(scanIt.hasNext()) {
+      Tuple outTuple = new VTuple(outColumnNum);
```

Comment:
It is actually for every row. While condition is just for check if there is next row, and every while loop will be just returned for each tuple.